JamfSync/Multipart/AmazonS3.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Copyright 2024, Jamf
+//
+
+import Cocoa
+import CryptoKit
+import Foundation
+
+
+class CompleteMultipart: NSObject {
+    var partNumber: Int
+    var eTag: String
+    
+    init(partNumber: Int, eTag: String) {
+        self.partNumber = partNumber
+        self.eTag = eTag
+    }
+}
+var partNumberEtagList = [CompleteMultipart]()
+
+struct Chunk {
```

Comment:
I might be misunderstanding the purpose of this struct. Does a `Chunk` not represent a literal chunk of data being sent? I'm confused why all the fields are `static`